docs/application/web/guides/app-management/app-management-cover.md

```diff
@@ -0,0 +1,28 @@
+# Application Management
+
+The application management features cover the application models available for Web applications. They describe the application life-cycle and resources, and the communication methods available for the application.
+
+You can use the following application management features in your Web applications:
+
+- [Applications](./applications.md)  
+You can manage applications and get information about them. You can also handle events and enable an application to run in the background.
+
+ Tizen provides various application models to allow you to create applications targeted for specific tasks. You can create UI applications to run on the foreground, and service applications to run in the background. The service application is used to perform tasks that need to run periodically or continuously, but do not require any user interaction.
```

Comment:
minor) This paragraph is sub item of the list, under `Application`. However it is out of the list line now.

Reply:
I will remove app-management-cover.md but applications.md will substitue this page's role.